bool boxMissing
bool penMissing
bool smfPrbtPickingBox
bool smfPrbtPickingPen
bool smfPrbtPlacingBox
bool smfPrbtPlacingPen
bool smfPrbtAtHome